proto/pdpb.proto

```diff
@@ -440,6 +447,12 @@ message StoreStats {
     uint64 keys_read = 14;
     // Actually reported time interval
     TimeInterval interval = 15;
+    // CPU usages of threads in the store
+    map<string, RollingStats> cpu_usages = 16;
```

Comment:
I think it is better to use total statistics of all threads for PD. 

For thread's statistics, we can send them to Prometheus.

Reply:
Maybe the total statistics of all threads is enough for PD now, but actually, I want to divide all threads into several categories, such as grpc, rocksdb, raft, etc., so PD can use this these diverse information to make intelligent decisions in the future.